lib/python/treadmill_aws/bootstrap/node/aws.linux/init1/registry.yml

```diff
@@ -0,0 +1,43 @@
+command: |
```

Comment:
should it be init1 or init service? init1 have dependency on server presence registered in zk. i do not think this is the case here, docker registry runs independently from everything else, so i would argue it is init service, not init1.